test/e2e/clusternetworkpolicy_test.go

```diff
@@ -0,0 +1,559 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	secv1alpha1 "github.com/vmware-tanzu/antrea/pkg/apis/security/v1alpha1"
+	. "github.com/vmware-tanzu/antrea/test/e2e/utils"
```

Comment:
:) I would try to avoid controversial usage: https://github.com/golang/go/issues/29326
Also the common review comment doesn't suggest using this unless you meet this situation:
https://github.com/golang/go/wiki/CodeReviewComments#import-dot
It's subconscious for Go developers to think a function/variable that doesn't have package qualifier is in its own package. This usage broke it and made it a little harder to find the source of the function/variable.

Reply:
Appreciate your suggestion on this. Will refactor this part when we address #893 Thanks